mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1846,6 +1849,24 @@ impl ElectrumClient {
         rpc_func_from!(self, server_address, "server.version", client_name, protocol_version)
     }
 
+    /// https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-headers-subscribe
+    pub fn get_block_count_from(&self, server_address: &str) -> RpcRes<u64> {
+        Box::new(
+            rpc_func_from!(self, server_address, BLOCKCHAIN_HEADERS_SUB_ID)
+                .map(|r: ElectrumBlockHeader| r.block_height()),
+        )
+    }
+
+    /// https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-block-headers
+    pub fn get_block_headers_from(
+        &self,
+        server_address: &str,
+        start_height: u64,
+        count: NonZeroU64,
+    ) -> RpcRes<ElectrumBlockHeadersRes> {
+        rpc_func_from!(self, server_address, "blockchain.block.headers", start_height, count)
```

Comment:
My idea was that the whole function signature `pub fn get_block_count_from(&self, server_address: &str)` gives this meaning, if your read the name with the input it would be `get_block_count_from_server_address` :)

Same for block headers function, `get_block_headers_from_server_address/start_height/count` . What do you think, can `start_height` and `count` have a better name e.g. `from_height`, `num_of_blocks`?

Reply:
ok great, I like `from_height` and `count`